clang-tools-extra/test/clang-tidy/checkers/bugprone/incorrect-enable-shared-from-this.cpp

```diff
@@ -0,0 +1,180 @@
+// RUN: %check_clang_tidy -std=c++11 %s bugprone-incorrect-enable-shared-from-this %t
```

Comment:
Note: you could use -std=c+=11-or-later to fix problems with check in more modern C++